modules/op-configuring-buildah-to-use-build-user.adoc

```diff
@@ -10,11 +10,11 @@ You can define a Buildah task to use the `build` user with user id `1000`.
 
 .Procedure
 
-. Create a copy of the `buildah` cluster task as an ordinary task.
+. Create a copy of the `buildah` task, which is provided in the `openshift-pipelines` namespace, modified to be an ordinary task.
```

Comment:
I don't understand `modified to be an ordinary task` - what do you want to say here?